src/System.IO.FileSystem.Watcher/src/System/IO/FileSystemWatcher.Linux.cs

```diff
@@ -455,6 +458,8 @@ private void ProcessEvents()
                     WatchedDirectory previousEventParent = null;
                     uint previousEventCookie = 0;
 
+                    const Interop.libc.NotifyEvents AddMaskFilters = Interop.libc.NotifyEvents.IN_CREATE | Interop.libc.NotifyEvents.IN_MOVED_TO;
+
```

Comment:
I'm still not clear why this isn't defined just before it's used below.  But not a big deal.